traveler-swift-core/TravelerKit/Models/WishlistResultError.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  WishlistResultError.swift
+//  TravelerKit
+//
+//  Created by Omar Padierna on 2019-09-03.
+//  Copyright © 2019 Ata Namvari. All rights reserved.
+//
+
+import Foundation
+/// Errors that can occur upon fetching the Wishlist
+public enum WishlistResultError: Error {
+    /// The traveler is not identified yer. Developer must call `Traveler.identify()` before attempting to fetch the wishlist
+    case unidentifiedTraveler
```

Comment:
There are a number of unidentifiedTraveler errors....can we consolidate them into one error type? The answer may be no but I think it deserves some thought